docs/photoniq/prerendering/manage-origins/view-404-identifiers.md

```diff
@@ -0,0 +1,41 @@
+---
+sidebar_position: 30
+title: View 404 Identifiers
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+You have several options for viewing Dynamic Prerendering 404 identifiers currently set on your origins. The 404 identifier is set when an origin is created. It signals to the service that the page is a 404 and should not be rendered.
+
+- **404 Identifier tab** - In the 404 Identifier tab in the web console, you can view information about the 404 identifier assigned to each origin. To add or update 404 identifiers, contact Macrometa Support.
+  ![Prerendering 404 Identifier Tab](/img/prerendering/404-identifier.png)
+- **REST API** - Macrometa provides API calls that allow you to view origin settings, including 404 identifiers, for all origins or for one origin.
+
+## View 404 Identifiers
+
+<Tabs groupId="operating-systems">
+<TabItem value="console" label="Web Console">
+
+Follow these instructions to view 404 identifiers in the web console.
+
+1. Log in to your Dynamic Prerendering dashboard.
+2. Click **404 Identifier**.
+
+</TabItem>
+<TabItem value="api" label="REST API">
+
+In the API, 404 identifier information is included in the origin settings API. Use our interactive API Reference with code generation in 18 programming languages to:
+
+- [Get origins](https://www.macrometa.com/docs/apiPrerendering#/paths/api-prerender-v1-origins/get)
+- [Get an origin](https://www.macrometa.com/docs/apiPrerendering#/paths/api-prerender-v1-origins-origin/get)
+
+</TabItem>
+</Tabs>
+
+## 404 Identifier Fields
+
+Dynamic Prerendering provides the following fields in the 404 Identifier tab. Field names in the web console are different than those in the API responses. Refer to the API endpoint documentation for a full explanation of the API response.
+
+- **Origin URL** - Full URL of the origin.
+- **Identifier String** - Text that signals the prerendering bot to ignore the page.
```

Comment:
Here is the same idea, we set the response status code to 404 when we find this `Identifier String` in the rendered page.

Reply:
What should the exact wording here?

```
String that signals the bot to set the response code to 404.
```
?

Reply:
Yes, something like:
`Text that signals the prerendering bot to set the response code to 404.`

Reply:
Updated